include/ProgramRepresentation/ProgramVariable.h

```diff
@@ -78,4 +78,4 @@ class ProgramVariable {
     bool isIdentifier() const;
 };
 
-#endif
```

Comment:
If you want all files to end with newlines, it would be a good idea to make a PR that does that uniformly for all files in the codebase and nothing else, to avoid cluttering other reviews like this one.

Reply:
(I don't care either way, but would prefer it to be consistent throughout the codebase.)